crates/interpreter/src/valid.rs

```diff
@@ -764,17 +810,60 @@ impl<'a, 'm> Expr<'a, 'm> {
         Ok(())
     }
 
-    fn br_label(&self, l: LabelIdx) -> Result<ResultType<'m>, Error> {
+    fn stitch(&mut self, branch: &SideTableBranch) {
+        let current_parser = self.parser.save().as_ptr() as isize;
+        let SideTableBranch { parser, side_table_len } = branch;
+        let entry = &mut self.side_table[*side_table_len];
+        if entry.is_none() {
+            *entry = Some(SideTableEntryView {
+                delta_ip: (current_parser - parser.as_ptr() as isize) as i32,
+                delta_stp: 0,
```

Comment:
`current.side_table - branch.side_table`

Reply:
`delta_stp` can be much larger than expected, and leads to [the error here](https://github.com/zhouwfang/wasefire/blob/382e5737b2b674a2ded1e864f29f8a4941cd1f9b/crates/interpreter/src/bit_field.rs#L32). We would need to make the masks larger, and I plan to do it in another PR.

Reply:
I added the computation for `delta_stp` in this PR which requires modifications to the masks.